libs/pageserver_api/src/config.rs

```diff
@@ -401,10 +413,12 @@ impl Default for ConfigToml {
             ephemeral_bytes_per_memory_kb: (DEFAULT_EPHEMERAL_BYTES_PER_MEMORY_KB),
             l0_flush: None,
             virtual_file_io_mode: None,
-            server_side_batch_timeout: DEFAULT_SERVER_SIDE_BATCH_TIMEOUT
-                .map(|duration| humantime::parse_duration(duration).unwrap()),
             tenant_config: TenantConfigToml::default(),
             no_sync: None,
+            page_service_pipelining: Some(PageServicePipeliningConfig {
```

Comment:
Can we enable this in a few test unit/python to get some rudimentary continuous test coverage?

Reply:
> Can we enable this in a few test unit/python to get some rudimentary continuous test coverage?

I don't get the question. If the default is `Some` here, so, we have the entire test suite as coverage.

Reply:
I assumed you won't merge it with pipelining enabled. If that's the case, I am suggesting to parametrize some existing python tests to use pipelining (we'd also have to tweak the compute config).